app/views/ProfileView/index.tsx

```diff
@@ -274,8 +274,19 @@ const ProfileView = ({ navigation }: IProfileViewProps): React.ReactElement => {
 
 	useFocusEffect(
 		useCallback(() => {
-			reset();
-		}, [])
+			if (user) {
```

Comment:
This is adding reactivity, but not fixing the actual issue.
The issue is after a submit, the values the user just entered are cleared.
That's still going to happen, but after the new user state gets to the app through websocket and redux, it's going to apply the new value on a re-render.

What you really want to do is to change the way we handle the form after the submit.

This reactivity also introduces unexpected behaviors.
Imagine you're changing something on your profile and something changes your user, like user status (a tab switch on web).
It's going to trigger reset and the current data is lost.
This is quite sensitive if you're changing your password, for example.